docs/guides/create-new-capability.md

```diff
@@ -34,8 +34,10 @@ Occasionally it makes sense to put multiple use-cases into a single profile file
 -->
 
 :::tip Scoped profiles
+
```

Comment:
Consider dropping these changes so it doesn't conflict with #95.

Reply:
And hotfix into `main`? I'd rather keep the changes, but you're right they don't belong in this PR